### PR TITLE
Added bower package file

### PR DESCRIPTION
**Pull request by gabrielrinaldi**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+language: sass
+rvm:
+  - "2.1.2"
+before_install:
+ - gem install sass rake
+script:
+ - rake test
```
Comment (chris-pearce) [inline .travis.yml:7]: https://www.youtube.com/watch?v=PXatLOWjr-k

**Comment by chris-pearce** [comment]
LGTM, @gabrielrinaldi @kllevin lets catch up your Mon and go over this so I can understand it better. @gabrielrinaldi can't merge cause Travis is upset: https://www.youtube.com/watch?v=PXatLOWjr-k.

**Comment by gabrielrinaldi** [comment]
It fails because Scally is not ready yet. Monday sounds good to me.

**Comment by chris-pearce** [comment]
Sound.

**Pull request merged**